Timeline.cpp

```diff
@@ -18,6 +21,23 @@ TweenDuino::Timeline::Timeline():
     initialized(false),
     lastUpdateTime(0) {}
 
+void TweenDuino::Timeline::wipe(){
+	totalDuration = 0;
+    totalTime = 0;
+    startTime = 0;
+    completed = false;
+    initialized = false;
+    lastUpdateTime = 0;
+	
+	for (int i = 0; i < TWEEN_TIMELINE_SIZE; i++) {
+        if (tweens[i].tween) {
+			delete tweens[i].tween;
+			tweens[i].tween = nullptr;
+		}
+    }
+}
+
+
```

Comment:
Thanks, really handy!

If you could, please do a pass on the indents.  There is a mixture of tabs and spaces; the library uses spaces for indents instead of tabs.